cxx-checks/src/main/java/org/sonar/cxx/checks/FunctionComplexityCheck.java

```diff
@@ -46,7 +47,8 @@
   offset = "10min",
   effortToFixDescription = "per complexity point above the threshold")
 public class FunctionComplexityCheck extends SquidCheck<Grammar> {
-  private static final int DEFAULT_MAX = 10;
+
+  private static final int DEFAULT_MAX = 8;
```

Comment:
Yes that is the drawback. You have to create a custom rule from the template: http://docs.sonarqube.org/plugins/servlet/mobile#Rules-RuleTemplatesandCustomRules.

Will set value back to 10. Any other opinions e.g. @Bertk. 

> Am 30.06.2015 um 21:50 schrieb Jorge Costa <notifications@github.com>:
> 
> In cxx-checks/src/main/java/org/sonar/cxx/checks/FunctionComplexityCheck.java:
> 
> > @@ -46,7 +47,8 @@
> >    offset = "10min",
> >    effortToFixDescription = "per complexity point above the threshold")
> >  public class FunctionComplexityCheck extends SquidCheck<Grammar> {
> > -  private static final int DEFAULT_MAX = 10;
> > +
> > +  private static final int DEFAULT_MAX = 8;
> Is it not more difficult for end user to use? It would be nice that the plugin would already define the rules and the default threadsholds ready to use. Using templates likely no one will notice them
> 
> —
> Reply to this email directly or view it on GitHub.
>